### ISSUE TITLE
[Enhancement] In-App Review

### ISSUE THREAD

**Issue by mariohik** [xamarin/Essentials#1389]
## Summary
Now that iOS and Android have revisions to the app, it would be nice to have Xamarin.Forms. Users can give a comment without leaving the application.
[Apple Store In-App Review](https://developer.apple.com/app-store/ratings-and-reviews/)
[Google Play In-App Review API](https://developer.android.com/guide/playcore/in-app-review)
[Integrate In-App Reviews for Java](https://developer.android.com/guide/playcore/in-app-review/kotlin-java#java)

## Intended Use Case
Prompt users to submit Play Store/Apple Store ratings and reviews without the inconvenience of leaving your app or game.

**Comment by jamesmontemagno** [comment]
This is a tricky one only because Android is a dependency that would need to be brought in and then couldn't be used on other app stores.

https://github.com/jamesmontemagno/StoreReviewPlugin is my plugin today that handles mac/ios/uwp for pop up and has other helpers for Android.

**Comment by mattleibow** [comment]
Yeah. Things interacting with stores on Android are touch-and-go. Things like version update checks are also very hard.

**Comment by mariohik** [comment]
[NuGet XamarinPlayCoreUpdater](https://github.com/PatGet/XamarinPlayCoreUpdater)

**Comment by jfversluis** [comment]
I think at this point it's safe to say that this won't be making it's way into Xamarin.Essentials anymore. Keep your eye on .NET MAUI for all new development! Thanks for all input and efforts here.
